src/CodeMaid.config

```
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+    <configSections>
+        <sectionGroup name="userSettings" type="System.Configuration.UserSettingsGroup, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" >
+            <section name="SteveCadwallader.CodeMaid.Properties.Settings" type="System.Configuration.ClientSettingsSection, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" />
+        </sectionGroup>
+    </configSections>
+    <userSettings>
+        <SteveCadwallader.CodeMaid.Properties.Settings>
+            <setting name="Cleaning_UpdateFileHeader_HeaderUpdateMode" serializeAs="String">
+                <value>1</value>
+            </setting>
+            <setting name="Cleaning_UpdateFileHeaderCSharp" serializeAs="String">
+                <value>// Copyright © 2017 - 2022 Chocolatey Software, Inc
+// Copyright © 2011 - 2017 RealDimensions Software, LLC
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// 
+// You may obtain a copy of the License at
+// 
+// 	http://www.apache.org/licenses/LICENSE-2.0
+// 

```

```suggestion
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
//
// You may obtain a copy of the License at
//
// 	http://www.apache.org/licenses/LICENSE-2.0
//
```

I know @gep13 will want these trailing spaces removed because his Visual Studio extension will do it every time for him.

Also: What is CodeMaid? If I were to guess, based upon the contents of this file, it's some sort of code hygiene tool, but I don't recall having heard of it before.

---

This one was actually not supposed to be checked in, that was a mistake.

It is the configuration file I am using for the Visual Studio extension of Codemaid, which is a utility to beautify, reorder and basically clean up source code files.

Even if including this file, this is the license of the application itself, not our so removing the newlines here would probably not be correct.

Anyways, I'll get the file removed.

---

I have removed this file now.